### PR TITLE
added correct version

### PR DESCRIPTION
**Pull request by flekros**

### CODE REVIEW

File: src/main/java/core/basesyntax/HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String stringa;
+    private int variable;
```
Comment (d1sam) [inline src/main/java/core/basesyntax/HelloWorld.java:9]: Please think about names of these varaibles.

File: src/main/java/core/basesyntax/model/Cat.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    @Override
+    public String toString() {
+        return "Cat{"
+                + "age="
+                + age + ", name='"
+                + name + '\''
+                + ", ownerName='"
+                + ownerName
+                + '\''
```
Comment (d1sam) [inline src/main/java/core/basesyntax/model/Cat.java:16]: Variable name should be on the same line with variable value for better readability.

**Review by d1sam — CHANGES_REQUESTED** [review]
Nice try, but you should review variables' names.

File: src/main/java/core/basesyntax/model/Cat.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    @Override
+    public String toString() {
+        return "Cat{"
+                + "age=" + age
+                + ", name='" + name
+                + '\''
+                + ", ownerName='" + ownerName
+                + '\''
+                + '}';
+    }
```
Comment (kshuryhin) [inline src/main/java/core/basesyntax/model/Cat.java:17]:
```suggestion
  @Override
public String toString() {
  return "Cat{"
      + "age=" + age
      + ", name='" + name + '\''
      + ", ownerName='" + ownerName + '\''
      + '}';
}
```
Regarding string concatenation, I would say we can do that in a more readable way, not cluttering our code with unnecessary whitespaces and indentations.

**Review by kshuryhin — APPROVED** [review]
Good try, minor comment regarding string concatenation, all previous comments are resolved.

**Pull request closed without merging**